openfl/component/director/director.py

```diff
@@ -6,50 +6,279 @@
 import asyncio
 import logging
 import time
-import typing
 from collections import defaultdict
 from pathlib import Path
+from typing import Iterable
+from typing import List
+from typing import Union
 
+from openfl.component import Aggregator
 from openfl.federated import Plan
 from openfl.protocols import director_pb2
+from openfl.transport import AggregatorGRPCServer
 from openfl.utilities.workspace import ExperimentWorkspace
 
 logger = logging.getLogger(__name__)
 
 ENVOY_HEALTH_CHECK_PERIOD = 60  # in seconds
 
 
+class Status:
+    """Experiment's statuses."""
+
+    PENDING = 'PENDING'
+    FINISHED = 'FINISHED'
+    IN_PROGRESS = 'IN_PROGRESS'
+    FAILED = 'FAILED'
+
+
+class Experiment:
+    """Experiment class."""
+
+    def __init__(
+            self, *,
+            name: str,
+            data_path: Union[Path, str],
+            collaborators: List[str],
+            sender: str,
+            init_tensor_dict: dict,
+            plan_path: Union[Path, str] = 'plan/plan.yaml',
+            aggregator_grpc_server: AggregatorGRPCServer = None,
```

Comment:
It is created inside and looks like we shouldn't pass it.

Reply:
I thought it would be useful when we start restoring active experiment after director restart. but I agree that it could be deleted for now

Reply:
Done